app/controllers/concerns/hyrax/collections_controller_behavior.rb

```diff
@@ -103,6 +103,8 @@ def create
       @collection.attributes = collection_params.except(:members)
       @collection.apply_depositor_metadata(current_user.user_key)
       add_members_to_collection unless batch.empty?
+      @collection.visibility = Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PRIVATE
```

Comment:
Should this be...
```
@collection.visibility = Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PRIVATE unless @collection.discoverable?
```
Otherwise, aren't you setting all collections to private?  Am I missing something?